### DISCUSSION TITLE
How are the standard modules validated?

### DISCUSSION THREAD

**Discussion by TriCogz** [synthetichealth/synthea#1420] [General]
Hi,
How are the standard modules validated? Were these converted/made up/MD standardized? Just interested. 
Thanks

**Comment by jawalonoski** [comment]
This is a long topic meriting significant discussion.

However, I will say that different modules are validated to varying degrees depending on their intended use. There is a spectrum: some very rigorously, some placeholders. Often merely looking at the module can give you a hint on where it is in that range.

We use a combination of clinical inputs, clinical practice guidelines, standards of care, and other documents as a starting point, and have review of:
1. The module
2. The patients that the module produces
3. The population as a whole that the module produces

We have written several papers that describe the process, and we invite you to look at these.

> Jason Walonoski, Mark Kramer, Joseph Nichols, Andre Quina, Chris Moesel, Dylan Hall, Carlton Duffett, Kudakwashe Dube, Thomas Gallagher, Scott McLachlan, Synthea: An approach, method, and software mechanism for generating synthetic patients and the synthetic electronic health care record, Journal of the American Medical Informatics Association, Volume 25, Issue 3, March 2018, Pages 230–238, https://doi.org/10.1093/jamia/ocx079

> Walonoski J, Klaus S, Granger E, Hall D, Gregorowicz A, Neyarapally G, Watson A, Eastman J. Synthea™ Novel coronavirus (COVID-19) model and synthetic data set. Intelligence-Based Medicine. 2020 Nov;1:100007. https://doi.org/10.1016/j.ibmed.2020.100007

> Walonoski J, Hall D, Bates KM, Farris MH, Dagher J, Downs ME, Sivek RT, Wellner B, Gregorowicz A, Hadley M, Campion FX, Levine L, Wacome K, Emmer G, Kemmer A, Malik M, Hughes J, Granger E, Russell S. The “Coherent Data Set”: Combining Patient Data and Imaging in a Comprehensive, Synthetic Health Record. Electronics. 2022; 11(8):1199. https://doi.org/10.3390/electronics11081199
